common/src/main/java/org/mvndaemon/mvnd/common/Environment.java

```diff
@@ -42,16 +44,20 @@
  */
 public enum Environment {
 
+    /**
+     * Print the completion for the given shell to stdout. Only <code>--completion bash</code> is supported at this time.
+     */
+    COMPLETION(null, null, null, OptionType.STRING, Flags.OPTIONAL, "mvnd:--completion"),
     /**
      * Delete log files under the <code>mvnd.registry</code> directory that are older than <code>mvnd.logPurgePeriod</code>
      */
-    PURGE(null, null, null, OptionType.VOID, Flags.OPTIONAL, "--purge"),
+    PURGE(null, null, null, OptionType.VOID, Flags.OPTIONAL, "mvnd:--purge"),
     /** Prints the status of daemon instances registered in the registry specified by <code>mvnd.registry</code> */
-    STATUS(null, null, null, OptionType.VOID, Flags.OPTIONAL, "--status"),
+    STATUS(null, null, null, OptionType.VOID, Flags.OPTIONAL, "mvnd:--status"),
     /** Stop all daemon instances registered in the registry specified by <code>mvnd.registry</code> */
-    STOP(null, null, null, OptionType.VOID, Flags.OPTIONAL, "--stop"),
+    STOP(null, null, null, OptionType.VOID, Flags.OPTIONAL, "mvnd:--stop"),
     /** Use one thread, no log buffering and the default project builder to behave like a standard maven */
-    SERIAL("mvnd.serial", null, Boolean.FALSE, OptionType.BOOLEAN, Flags.OPTIONAL, "--serial"),
+    SERIAL("mvnd.serial", null, Boolean.FALSE, OptionType.BOOLEAN, Flags.OPTIONAL, "mvnd:-1", "mvnd:--serial"),
```

Comment:
@gnodet there was no short option for --serial here. I have put `-1` there. I am open to discuss if you do not like it.

Reply:
That's fine.